src/utils/api.js

```diff
@@ -43,3 +51,9 @@ export function postComment(id, post) {
     .then((response) => {})
     .catch((err) => {});
 }
+
+export function getCategories() {
+  return server.get(`/categories`).then((response) => {
+    return response.data.categories;
+  });
```

Comment:
Good to see a new function for this api call, the name `getCategories` is a well named function name too, well done